Resources/Maps/Corvax/corvax_chloris.yml

```
@@ -298772,7 +298772,7 @@ entities:
       setupAvailableNetworks:
       - SurveillanceCameraSecurity
       nameSet: True
-      id: Предгенпоп
+      id: Пред общ. блок
```

```suggestion
      id: Прих общ. блок
```
Ну типо прихожая или проходная или пропускная